sdk/notificationhubs/notification-hubs/src/serializers/notificationHubJobSerializer.ts

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getDateOrUndefined, getInteger, getString, getStringOrUndefined } from "../utils/xmlUtils";
+
+export enum NotificationHubJobType {
```

Comment:
same question about possibility of conversion to a union of strings.

Reply:
On the back end, it's represented as a number.  Is it worth making it a string then?

Reply:
Fixed